src/main/java/cz/smarteon/loxone/user/UserGroupListValue.java

```diff
@@ -0,0 +1,37 @@
+package cz.smarteon.loxone.user;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cz.smarteon.loxone.message.LoxoneValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * {@link LoxoneValue} where the JSON string is expected to encode a list of user groups.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter
+@AllArgsConstructor
+public class UserGroupListValue implements LoxoneValue {
+
+    private final List<UserGroup> userGroups;
+
+    @JsonCreator
+    public UserGroupListValue(String userGroups) throws JsonProcessingException {
+        List<UserGroup> list = new ArrayList<>();
```

Comment:
this is not how deserialization should be done - at least ObjectMapper is heavyweight object, creating it over and over with each instance of some class my cause significant performance issues.

Second, let's find out how the whole message looks like and leverage Jackson capabilities and try to avoid calling objectmapper in the constrcutor at all.